README.md

```diff
@@ -34,21 +56,106 @@ Create a json config file with `branch-config` and/or `wctflags` keys:
 `tattoo_config.json`:
 ```
 {
-  "branch-config": {
-    "polymer" : "Polymer/polymer#lazy-register-extends",
-    "paper-button" : "PolymerElements/paper-button#master"
-  },
-  "wctflags": ["-b", "canary"]
+  "test": [
```

Comment:
Updated the section around the config file to make it clearer and removed that bad reference you pointed out.  Fixed in f857a44517b976dd6ca994277a5d1b95d79c1998